tests/kernel/arch_interface/misc/src/main.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include <ztest.h>
+
+#define SHIFT_BITS	10
+#define TEST_ROUND	(1 << SHIFT_BITS)
+
+/**
+ * @brief Test how much time arch_nop() costs
```

Comment:
It better to replace "costs" with "takes".

Reply:
Correct it, thanks!